Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -3415,8 +3415,8 @@ - (void)dismissTaskViewController:(ORKTaskViewController *)taskViewController re
 - (BOOL)taskViewController:(ORKTaskViewController *)taskViewController hasLearnMoreForStep:(ORKStep *)step {
     NSString *task_identifier = taskViewController.task.identifier;
 
-    return ([step isKindOfClass:[ORKInstructionStep class]]
-            && NO == [@[AudioTaskIdentifier, FitnessTaskIdentifier, GaitTaskIdentifier, TwoFingerTapTaskIdentifier, NavigableOrderedTaskIdentifier, NavigableLoopTaskIdentifier] containsObject:task_identifier]);
+    return (/*[step isKindOfClass:[ORKInstructionStep class]]
+            &&*/ NO == [@[AudioTaskIdentifier, FitnessTaskIdentifier, GaitTaskIdentifier, TwoFingerTapTaskIdentifier, NavigableOrderedTaskIdentifier, NavigableLoopTaskIdentifier] containsObject:task_identifier]);
```

Comment:
For testing, i made every step carry a learn more button. Once this is tapped the printing dialog appears. When we uncomment this line, only certain instruction steps will have a learn more button visible. I suggest we keep it as long as we test the printing feature and remove ist before this PR gets merged. I've added a //TODO: